src/types/organization.d.ts

```diff
@@ -1,10 +1,19 @@
+interface EmailDomains {
+  domain: string;
+  type:
+    | "verified"
```

Comment:
```suggestion
  type:
    | "backlisted"
    | "verified"
```

type can be "backlisted"

Reply:
Out of scope